examples/bandsplot-example.ipynb

```diff
@@ -107,7 +107,13 @@
     "    data3 = json.load(file)\n",
     "    \n",
     "with open('./data/Co_dos.json', 'r') as file:\n",
-    "    data4 = json.load(file)"
+    "    data4 = json.load(file)\n",
+    "    \n",
+    "with open('./data/band-converged-80.json', 'r') as file:\n",
```

Comment:
Do you still use this filename? We need to make the filename more clear. If you have change the JSON content to what you used for the example shown in the comment, you can call it `bands-silicon-example.json`.

Reply:
I removed this json data and use a modifed Si bandstructure json file.